src/wp-includes/sitemaps.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Sitemaps: Public functions
+ *
+ * This file contains a variety of public functions developers can use to interact with
+ * the XML Sitemaps API.
+ *
+ * @package WordPress
+ * @subpackage Sitemaps
```

Comment:
I thought at some point it was decided to not use `@subpackage` anymore, at least not for new code. Don't recall exactly, but we may want to check further and potentially remove these.

Reply:
According to the handbook it's fine for now:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#deprecated-tags